gsoc2018-bharat/src/pre-train.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import logging
+from gensim.models.word2vec import LineSentence
+from gensim.models import FastText
+from gensim.models import Word2Vec
+"""
+We are using the wrapper for FastText available in the Gensim package. FastText is an extension to the word2vec
+skipgram model, as it uses character n-grams to generate word embeddings. This allows for the model to create embeddings
+for out of vocabulary words with the help of previously identified n-gram embeddings.
+"""
+logging.basicConfig(format='%(levelname)s : %(message)s', level=logging.INFO)
+logging.root.level = logging.INFO
```

Comment:
Do you need this, I though you already had pre trained fast text vectors.

Reply:
Pre-trained embeddings for Word2Vec, FastText and GloVe are anyway available. We can use them. This script is to see how the training of these models works in the Gensim library.